### ISSUE TITLE
Clarify wording on "use of sexualized language or imagery"

### ISSUE THREAD

**Issue by seniorwrangler** [ContributorCovenant/contributor_covenant#255]
The phrase "use of sexualized language or imagery" is problematic because there are many situations where, depending to project context, the use of sexualized language or imagery may be appropriate.

Examples might be:

- A project dealing with sexual health
- A project dealing with gay rights
- A project dealing with adult humour / comedy

I suggest rewording this to either:
- "inappropriate use of sexualized language or imagery"
- "unjustified use of sexualized language or imagery"

However it is worded, I think the intention should be made clear that the purpose is not to prohibit the use of sexualized language or imagery entirely, but to prevent them from being used to harass individuals.

As background: I am a semi-prominent open source contributor, posting under an alternate account as I do not want my real name publicised. I am interested in adopting a CoC such as the Contributor Covenant for various projects if this issue can be addressed. Broadly, I identify as "Sex Positive" as defined here: http://www.millennialmanifesto.literallydarling.com/identifying-as-sex-positive/ so I will not adopt any CoC that, in my opinion, restricts self-expression regarding sexual issues.

**Comment by CoralineAda** [comment]
I appreciate you reaching out with these concerns, but I think that there is a clear difference between the contexts that you are pointing out and the intent of the language in the code of conduct. Contextually appropriate sexual content is not the same as sexualized language. And of course it is ultimately the role of project maintainers to clarify the rules around language usage, so of course in the case of a project dealing with sexual health no one would expect such a provision to be exercised simply for talking about the topic at hand.

**Comment by seniorwrangler** [comment]
Thanks for your quick reply!

I agree with with what you say, however the wording of the CoC does not currently seem to reflect this, as I read: "Examples of unacceptable behavior by participants include.... The use of sexualized language or imagery"

This reads as a explicit blanket ban on the use of sexual content, which I feel is a major problem and goes beyond the intended scope of the CoC (i.e. preventing harassment). 

It would be far better if the CoC made it clear that *appropriate* use of sexual content is acceptable. I don't think the discretion of project maintainers is sufficient here (since they would have to go against the explicit wording of the CoC), and I also would't want to encourage people to add exception / disclaimers / clarifying provisions (otherwise what is the point of encouraging a standardised CoC?)

**Comment by mprelude** [comment]
@seniorwrangler For something like this you're probably best off forking the CoC and replacing the clause with 'unwelcome sexual advances'. It's not 'one size fits all'.

**Comment by zaxeni** [comment]
Will the CoC apply to project using sexual name : https://github.com/edankwan/penis.js

**Comment by webmaven** [comment]
@seniorwrangler, I'll repeat what @CoralineAda said: sexualized is not the same as sexual. A blanket on on sexualized language and imagery is entirely appropriate.

@zaxeni, are you trolling? If the maintainer of that project wants to adopt the CoC, they can.

**Comment by beoran** [comment]
But this is exactly the problem. By using the unclear word 'sexualized' it creates confusion. What even is "sexualized language"? It's extremely vague, like most of the conde of conduct, by the way.  This part of the code of conduct also smacks to me of prudery. The problem is not sexual language, or sexualized language (whatever that even means). The problem is personal attacks, I.E. trolling.

**Comment by webmaven** [comment]
@beoran:
> By using the unclear word 'sexualized' it creates confusion. What even is "sexualized language"?

Since you find “sexualized” ambiguous and confusing (FTR, it simply means making something sexual that isn't inherently so, or at least more sexual than it is inherently), is “objectifying” a clearer label to you?

**Comment by beoran** [comment]
The explanation you propose makes the term rather pointless. By it, it
would be ok if I discussed something that is inherently sexual, like, say,
porn, no matter the circumstances.

As for objectification, what is objectifying speech anyway?

It seems to me that what is meant is that no one should make sexual
advances towards others, but if that is the case it should be stated as
such. Like it is now, it seems like discussing anything sexual is not
allowed as well.

Anyway, I think it is pointless to ban specific topics of speech, since it
is impossible to say what is appropriate or not without the context of the
project and it's participants.  Rather than even attempt to police speech
by topic, most of such specifics  could be replaced by "Stay on topic as
defined by the project maintainers."

**Comment by webmaven** [comment]
@beoran:
> As for objectification, what is objectifying speech anyway?

OK, I think we've crossed over into Sexism 101 territory here. I'm done.

**Comment by beoran** [comment]
No really, I am not a native English speaker, and I am not from the USA, so
I find this concept of objectification bizarre. Please do explain.

However, if you are not willing to provide reasonable arguments as I have
done, then yes, that is your loss.

**Comment by mprelude** [comment]
@beoran 

> No really, I am not a native English speaker, and I am not from the USA, so I find this concept of objectification bizarre. Please do explain. However, if you are not willing to provide reasonable arguments as I have done, then yes, that is your loss.

Objectification is removing human value from somebody and just talking about them as an object, for example talking about their body without considering their human worth. It's another term which would make no sense in the CoC since in advertising industry & many others, its absolutely rampant.

If I (as a native English speaker) could not categorise speech according to the CoC, how can people from around the world with different linguistic & cultural backgrounds possibly act with the knowledge that they are in compliance or not in compliance with it.

CoC maintainers need to consider practicality when determining language which preserves the concept of mens rea (this means that, whenever I say something, it should be obvious to me before I say it whether I'm complying with the CoC). It's improving but there's a long way to go before it's right.

**Comment by beoran** [comment]
Well, I don't like most advertising either, but it is the nature of the
medium that we don't get to know the actors involved.  When I see a baby in
a diapers package I realise it is there to try and appeal to my emotions so
I will think less and buy more. But I realise this baby too is a person. I
sometimes wonder how they are doing now.

I do like what you are proposing about mens rea. I have seen alternative
codes of conduct that simply banned any communication done in bad faith,
that is with the purpose to cause harm. Which I think to be a nice idea,
but sometimes hard to decide on...

**Comment by lukego** [comment]
If I may piggy-back a question on this issue: Is _Jungfrau_ an acceptable name for a project according to this CoC?

On the one hand in English Jungfrau is a mountain in Switzerland, in the Jungfrau region, and I intend to use this alpine imagery to represent the project. This seems appropriate to me because English is the language of the project and the Jungfrau region is where I live.

On the other hand in German Jungfrau means Virgin and although the contemporary usage seems to be gender-neutral and non-sexual (as in English) I am told that the archaic usage (e.g. middle ages) would carry connotations of women as commodities, human sacrifices, etc and may be provocative to German feminists.

So! My own take is that the name Jungfrau is non-sexualized and consistent with the CoC but I would appreciate feedback on whether that is the right interpretation. I have to be cautious since I am the person who proposed this CoC for our project and volunteered to enforce it.

**Comment by mprelude** [comment]
@lukego Someone would have to be actively looking to be offended in order to bring up the archaic meaning of a word to be offended by it. I'd say you're fine.

**Comment by skylarmt** [comment]
I think PR https://github.com/ContributorCovenant/contributor_covenant/pull/580 might fix this issue, since it would prevent simple misunderstandings of a situation from being in violation of this clause of the CoC.

**Comment by beoran** [comment]
I think not. This CoC suffers from being both to vague in some aspects as to leave too much leeway to maintainers or outsiders, and at the same time is too specific, like in this case. In other words it's a leaky blacklist.

In my own document, WELCOME, I state that "off topic" communication is not allowed, and have a separate document with the points and project (GOALS) that are considered "on topic". A whitelist, which is project-specific. Such an approach is far more flexible whilst keeping the main document simple.

**Comment by camerons** [comment]
Improving the clarity around the term "sexualized language of imagery" is something that I feel would be very valuable. It is a topic we have struggled with in the Open Source Geospatial Community.
We consumed significant bandwidth [discussing](https://lists.osgeo.org/pipermail/discuss/2015-June/thread.html) whether the picture of a girl band and a Salvador Dalí painting should be removed as per our [CoC](http://www.osgeo.org/code_of_conduct/). No clear resolution was found. Excessive bandwidth on such topics tends to result in active contributors silently disengaging as they can't keep up. Clear guidelines would help.

While developing our CoC, the [following suggestion](https://lists.osgeo.org/pipermail/conference_dev/2015-January/002849.html) resonated with me, and influenced our thinking:
> Would it be appropriate for a presenter to include a [sexualized] image from a main stream media commercial?
> On 15/1/15 9:21 am, Eli Adam wrote:
> > I think the issue here is relevancy.  Sexualized images aren't necessarily entirely prohibited.   Irrelevant sexualized images are entirely prohibited.


I suggest that we can make use of film classifications, which already have decades of experience in defining "sexualized images", and what is appropriate for different age brackets. Unfortunately, different countries appear to have different definitions. The [Australian film definitions](https://en.wikipedia.org/wiki/Australian_Classification_Board#Film_and_video_game_classifications) are good and something I'd suggest we could borrow from. I'd suggest that communication forums should align with the categorisation of "General Exhibition":

> General (G) – Contains material available for general viewing. This category does not necessarily designate a children's film or game. Although not mandatory at this category, the Board may provide consumer information. Consumer advice at G classification usually relates to impacts on very young children. The content is very low in impact.
> * Themes should have a very low sense of threat or menace, and be justified by context.
> * Violence should have only a low sense of threat or menace, and be justified by context.
> * Sexual activity should be very mild and very discreetly implied, and be justified by context.
> * Coarse language should be very mild and infrequent and justified by context.
> * Drug use should be implied only very discreetly, and be justified by context.
> * Nudity should be justified by context.
